src/Catalyst.Tools.KeyGenerator/Program.cs

```diff
@@ -128,12 +122,10 @@ private static void ParseCommand(string[] args)
             if (commandExists)
             {
                 var parserResult = Parser.Default.ParseArguments(commandArgs, parsedCommand.OptionType);
-                if (parserResult.Tag == ParserResultType.NotParsed)
-                {
-                    return;
-                }
+                if (parserResult.Tag == ParserResultType.NotParsed) return;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4612924)